datadog-monitors-monitor-handler/example_inputs/inputs_1_create.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think this can be removed? IIUC its an example input for the tests based on the template schema?